state_synchronizer/src/counters.rs

```diff
@@ -3,15 +3,43 @@
 
 use lazy_static;
 use metrics::OpMetrics;
-use prometheus::IntCounter;
+use prometheus::{IntCounter, IntGauge};
 
 lazy_static::lazy_static! {
     pub static ref OP_COUNTERS: OpMetrics = OpMetrics::new_and_registered("state_sync");
 }
 
+/// Number of sync requests sent from a node
+pub const REQUESTS_SENT: &str = "requests_sent";
```

Comment:
why sometimes defined as string and sometimes as IntCounter. Can we do one way for consistency?

Reply:
So the ones I defined as string are the ones that need additional formatting (like attaching peer ID to it when we increment). I can just hardcode the string when I increment, but thought it'd be better to declare it as a constant here so we can keep track of all the metrics we have related to state sync